main.py

```
@@ -1,26 +1,45 @@
+app = typer.Typer(
+    name="nanoagent", 
+    help="NanoAgent - 极简 Agent 框架", 
+    add_completion=False
+)
 
 
 @app.command()
-def run(task: str = typer.Argument(..., help="任务描述")) -> None:
+def run(
+    task: str = typer.Argument(..., help="任务描述"),
+    max_iterations: int = typer.Option(None, "--max", "-m", help="最大迭代次数（可选）")
```

Ruff (0.15.10)

[warning] String contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF001)